### PR TITLE
http json converter escaping and valid empty json conversion added 

### LINKED ISSUE #70: Broken Json for scenrios not already run
Json output is broken when selected scenrio has noch metrics inside metrics store 
grafik

### LINKED ISSUE #125: escape characters for json converter
https://github.com/abap-observability-tools/abap-metrics-provider/blob/main/src/zamp_converter/zcl_amp_conv_http_json.clas.abap

https://www.tutorialspoint.com/json_simple/json_simple_escape_characters.htm

### PR DESCRIPTION
**Pull request by JohannesKonings**
closes #125 
closes #70

### CODE REVIEW

File: src/zamp_converter/zcl_amp_conv_http_json.clas.abap [hunk 1]
```diff
@@ -34,7 +34,11 @@ CLASS zcl_amp_conv_http_json IMPLEMENTATION.
 
       LOOP AT metric_store ASSIGNING FIELD-SYMBOL(<metric>) WHERE metric_group = <metric_store_group>-metric_group.
 
-        converted_metrics = converted_metrics && |"{ <metric>-metric_key }" : { <metric>-metric_value },|.
+        DATA(metric_key) = <metric>-metric_key.
+
+        REPLACE ALL OCCURRENCES OF REGEX `(["\\\/])` IN metric_key WITH `\\$0`.
```
[the remark this answers is not in the document]
Comment (JohannesKonings) [reply src/zamp_converter/zcl_amp_conv_http_json.clas.abap:39]: Good hint. Maybe we can raise a good first issue to do that afterwards?

[the remark this answers is not in the document]
Comment (Goala) [reply src/zamp_converter/zcl_amp_conv_http_json.clas.abap:39]: Sure 👍

**Review by Goala — APPROVED** [review]
LGTM

File: src/zamp_converter/zcl_amp_conv_http_json.clas.abap (hunk 1 above)
[the remark this answers is not in the document]
Comment (JohannesKonings) [reply src/zamp_converter/zcl_amp_conv_http_json.clas.abap:39]: Issue created: https://github.com/abap-observability-tools/abap-metrics-provider/issues/133

**Pull request merged**